extensions/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/runtime/config/runtime/exporter/OtlpExporterTracesConfig.java

```diff
@@ -89,9 +91,9 @@ interface TrustCert {
         Optional<List<String>> certs();
     }
 
-    public static class Protocol {
+    class Protocol {
         public static final String GRPC = "grpc";
-        public static final String HTTP_PROTOBUF = "http/protobuf";
-        public static final String HTTP_JSON = "http/json";
+        public static final String HTTP_PROTOBUF = "http";
+        public static final String HTTP_JSON = "http+json";
```

Comment:
The value for HTTP_JSON is not defined yet, we should not create one that most likely will change.
The current JSON support is for local serialisation and storage, prior to send to the collector.

Reply:
Again, I'll let you and @radcortez decide :)

Reply:
I removed the JSON field completely